src/Microsoft.ML.Experimental/DataLoadSave/Database/DatabaseSource.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Data.Common;
+using Microsoft.ML.Runtime;
+
+namespace Microsoft.ML.Data
+{
+    public sealed class DatabaseSource
+    {
+        public DatabaseSource(DbProviderFactory providerFactory, string connectionString, string commandText = "")
```

Comment:
We probably don't want a default on `commandText` right? Normal operations wouldn't work without supplying a commandText.

Reply:
Yeah, that's probably right. It doesn't make sense to have a default if the user isn't overriding anything

Reply:
Fixed.